src/aeb_fate_generate_ops.erl

```diff
@@ -247,6 +247,12 @@ ops_defs() ->
     , { 'DBG_UNDEF',           16#b3,  false,    true,   true,    ?GAS(0),       [a, a],     dbg_undef,               {string, any},    none, "Debug Op: Undefine a variable. Args = {var_name, register}" }
     , { 'DBG_CONTRACT',        16#b4,  false,    true,   true,    ?GAS(0),          [a],  dbg_contract,                    {string},    none, "Debug Op: Name the current contract. Args: {contract_name}"}
 
+    , { 'BYTES_SPLIT_ANY',     16#b5,  false,    true,   true,   ?GAS(10), [a, a, a],     bytes_split_any,   {bytes, integer}, variant, "Arg0 := bytes_split_any(Arg1, Arg2), where a positive Arg2 is the length of the first chunk, and a negative Arg2 is the length of the second chunk. Returns None if byte array is not long enough."}
```

Comment:
Why arg2 has to be negative?

Reply:
It doesn't have to be negative, it _can_ be negative. The alternative would be a three argument function, I thought this was more compact.

Reply:
I misread, makes sense now